src/main/java/seedu/address/model/client/SuggestionType.java

```diff
@@ -18,6 +18,13 @@
     public static final String MESSAGE_CONSTRAINTS = "Suggestion type can only be of the following: "
             + BY_FREQUENCY + ", " + BY_AVAILABLE + ", " + BY_CONTRACT;
 
+    public static final String FREQUENCY_DESCRIPTION = "Frequency: Clients who were edited more recently will appear "
+            + "higher up on list.";
+    public static final String AVAILABLE_DESCRIPTION = "Availability: Only clients whose current time is 1800-2200H in "
+            + "their time zone (off work hours) will be shown.";
+    public static final String CONTRACT_DESCRIPTION = "Contract expiry date: Clients whose contracts are expiring will "
+            + "be shown first.";
```

Comment:
Maybe modify to be the same as `FREQUENCY_DESCRIPTION` which mentions higher up on the list.

Reply:
done